docs/lib/datastore/fragments/android/getting-started/50_initDataStore.md

```diff
@@ -0,0 +1,66 @@
+To initialize the Amplify Storage and Authentication categories, we are required to use the `Amplify.addPlugin()` method for each category we want.  When we are done calling `addPlugin()` on each category, we finish configuring Amplify by calling `Amplify.configure()`.
```

Comment:
doesn't look like we are adding authentication?

Reply:
this should probably be... "To initialize the Amplify DataStore category, ..."

Reply:
yeah, leftover from a copy and paste! :D

thanks for catching it, will fix